### ISSUE TITLE
[Feature Request] x-axis marker for daystart or individual points

### ISSUE THREAD

**Issue by AleksCee** [henrywhitaker3/Speedtest-Tracker#442]
First: it’s a very great project, yesterday I switched over to this from an other Docker-speedtest. Thanks for it!

**Describe the feature**
For better orientation in the graphs to add some marker lines on the x-axis like daystart at 0h or individual lines every x-hours....
perhaps automatically on the showing range. So if sowing days > than 2 weeks a marker for week-start, > than 30 a marker for months-start.....

Thanks, Alex

**Comment by henrywhitaker3** [comment]
Good idea, will be doing this, but can't promise when I'll have the time
